jito_protos/build.rs

```diff
@@ -14,6 +14,7 @@ fn main() {
                 "protos/shared.proto",
                 "protos/shredstream.proto",
                 "protos/trace_shred.proto",
+                "protos/deshred.proto",
```

Comment:
should check out your mev-protos branch in the protos repo so it compiles and commit the submodule hash to this

Reply:
Fixed in https://github.com/jito-labs/shredstream-proxy/pull/40/commits/9a21c9157006c7731a6a6dac16d2da8770e29c10